### DISCUSSION TITLE
Terminal output overlapping area below

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#2890] [Support] [answered]
Heyo! Is this something I can fix?
The terminal is overlapping the area underneath it as shown in the screenshot.
![20201219-182655](https://user-images.githubusercontent.com/24299563/102696750-71c82800-4228-11eb-8cbb-0e989cf771af.png)

**Comment by parkervcp** [comment, marked as answer]
This appears to be the same issue as here - https://github.com/pterodactyl/panel/issues/2799

**Reply by ghost** [reply]
Ah, thanks. I was able to fix this by simply upping my font size to 14 instead of 13. Though, wouldn't this be a bug for accessibility? I saw that someone shut that issue down since it's not important.
